modules/move-etcd-different-disk.adoc

```diff
@@ -0,0 +1,182 @@
+  name: 98-var-lib-etcd
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Make File System on /dev/sdb
+          DefaultDependencies=no
+          BindsTo=dev-sdb.device
+          After=dev-sdb.device var.mount
+          Before=systemd-fsck@dev-sdb.service
+
+          [Service]
+          Type=oneshot
+          RemainAfterExit=yes
+          ExecStart=/usr/lib/systemd/systemd-makefs xfs /dev/sdb
+          TimeoutSec=0
+
+          [Install]
+          WantedBy=var-lib-containers.mount
+        enabled: true
+        name: systemd-mkfs@dev-sdb.service
+      - contents: |
+          [Unit]
+          Description=Mount /dev/sdb to /var/lib/etcd
+          Before=local-fs.target
+          Requires=systemd-mkfs@dev-sdb.service
+          After=systemd-mkfs@dev-sdb.service var.mount
+
+          [Mount]
+          What=/dev/sdb
+          Where=/var/lib/etcd
+          Type=xfs
+          Options=defaults,prjquota
+
+          [Install]
+          WantedBy=local-fs.target
+        enabled: true
+        name: var-lib-etcd.mount
+      - contents: |
+          [Unit]
+          Description=Sync etcd data if new mount is empty
+          DefaultDependencies=no
+          After=var-lib-etcd.mount var.mount
+          Before=crio.service
+
+          [Service]
+          Type=oneshot
+          RemainAfterExit=yes
+          ExecCondition=/usr/bin/test ! -d /var/lib/etcd/member
+          ExecStart=/usr/sbin/setenforce 0
+          ExecStart=/bin/rsync -ar /sysroot/ostree/deploy/rhcos/var/lib/etcd/ /var/lib/etcd/
+          ExecStart=/usr/sbin/setenforce 1
+          TimeoutSec=0
+
+          [Install]
+          WantedBy=multi-user.target graphical.target
+        enabled: true
+        name: sync-var-lib-etcd-to-etcd.service
+      - contents: |
+          [Unit]
+          Description=Restore recursive SELinux security contexts
+          DefaultDependencies=no
+          After=var-lib-etcd.mount
+          Before=crio.service
+
+          [Service]
+          Type=oneshot
+          RemainAfterExit=yes
+          ExecStart=/sbin/restorecon -R /var/lib/etcd/
+          TimeoutSec=0
+
+          [Install]
+          WantedBy=multi-user.target graphical.target
+        enabled: true
+        name: restorecon-var-lib-etcd.service
+
+----
+
+.  Create the machine configuration by entering the following commands:
++
+[source,terminal]
+----
+$ oc login -u ${ADMIN} -p ${ADMINPASSWORD} ${API}
+... output omitted ...
+----
++
+[source,terminal]
+----
+$ oc create -f etcd-mc.yml
+machineconfig.machineconfiguration.openshift.io/98-var-lib-etcd created
+----
++
+[source,terminal]
+----
+$ oc login -u ${ADMIN} -p ${ADMINPASSWORD} ${API} [... output omitted ...] $ oc create -f etcd-mc.yml machineconfig.machineconfiguration.openshift.io/98-var-lib-etcd created
+----
++
+The nodes update and reboot. After the reboot completes, the following events occur:
```

Comment:
```suggestion
The nodes are updated and rebooted. After the reboot is completed, the following events occur:
```
A couple of minor suggestions here. Unless the nodes update and reboot themselves, I think we need to adjust the wording to indicate that they are updated and rebooted by some other entity.